src/dotnet/UI.Blazor/Components/Modal/DialogFrame.razor

```diff
@@ -1,19 +1,43 @@
+@using ActualChat.UI.Blazor.Services
 @namespace ActualChat.UI.Blazor.Components
+@implements IAsyncDisposable
 
-<ModalFrame Class="@Class" Title="@Title" HasHeader="HasHeader" HasCloseButton="HasCloseButton">
-    <div class="dialog-body">
-        @Body
-    </div>
-    <div class="dialog-buttons">
-        @Buttons
-    </div>
-</ModalFrame>
+<div @ref="_elementRef" tabindex="-1" autofocus>
```

Comment:
Ok, I guess I get the part on element kind now, but...

Reply:
we need `tabindex="-1"` to be able to focus an element